master/src/main/java/org/apache/celeborn/service/deploy/master/clustermeta/AbstractMetaManager.java

```diff
@@ -468,6 +468,12 @@ public void updatePartitionSize() {
     } else {
       estimatedPartitionSize = initialEstimatedPartitionSize;
     }
+
+    // Do not trigger update is estimated partition size value is unchanged
+    if (estimatedPartitionSize == oldEstimatedPartitionSize) {
+      return;
```

Comment:
will this cause skip informing to new joined workers?

Reply:
For newly registered workers we update the estimated partition size on `RegisterWorker` event itself.

https://github.com/apache/celeborn/blob/c0ca9523f9ca8db913a512521a9c3559d5ae19d5/master/src/main/java/org/apache/celeborn/service/deploy/master/clustermeta/AbstractMetaManager.java#L256

Reply:
Ha, it's okay then